src/Lower.cpp

```diff
@@ -808,7 +808,21 @@ class FindBuffers : public IRGraphVisitor {
     using IRGraphVisitor::visit;
 
     void visit(const Call *op) {
-        IRGraphVisitor::visit(op);
+
+        if (op->call_type == Call::Extern) {
```

Comment:
Why is this code necessary? It looks like it'll skip over input images that are only ever used in an extern call. The Variable visitor below skips over variables with no associated Parameter object anyway.